shell.nix

```diff
@@ -8,31 +8,31 @@ let
   inherit (gerbilPackages-unstable) gerbil-ethereum gerbil-poo;
   inherit (gerbil-support) gerbilLoadPath;
 in
-  pkgs.mkShell {
-    inputsFrom = [
-      skyprotocol
-    ];
-    buildInputs = with pkgs; (
-      skyprotocol.buildInputs ++
-      lib.optional ethereum go-ethereum ++
-      [ netcat go-libp2p-daemon ] # used by integration tests
-                                  # TODO: Save at compile time
-                                  # the path to the p2pd (go-libp2p-daemon) binary.
-                                  # The path is useful because it is a hash -
-                                  # so nix knows we depend on this exact version.
-      );
-    shellHook = ''
-      echo ${gerbil-poo.src}; echo ${pkgs.gerbilPackages-unstable.gerbil-poo.src} ; echo
-      echo ${skyprotocol.src}; echo ${pkgs.gerbilPackages-unstable.skyprotocol.src} ; echo ${pkgs.gerbilPackages-unstable.skyprotocol.src} ; echo ${pkgs.gerbil-support.gerbilPackages-unstable.skyprotocol.src} ; echo
-      echo ${toString skyprotocol.passthru.pre-pkg.gerbilInputs}
-      echo ${gerbil-poo}
+pkgs.mkShell {
+  inputsFrom = [
+    skyprotocol
+  ];
+  buildInputs = with pkgs; (
+    skyprotocol.buildInputs ++
+    lib.optional ethereum go-ethereum ++
```

Comment:
You can remove the go-lib2p2-daemon stuff — the new gerbil-libp2p won't depend on it.

Reply:
go-libp2p-daemon stuff is only the `go-ethereum` or `ethereum` as well?

Reply:
Oh, sorry, my bad, I wanted to remove `shell.nix`, `pkgs,nix` and `default.nix` if that's okay with you and depend only on flake setup. I'm not eager to support 2 installation methods, but I can, if that's what's needed. @fare 

I haven't touched the existing nix expressions, except for auto-formating which my editor does.